### PR TITLE
Added a warning in the config file about Vulkan being obsolete and buggy.

### PR DESCRIPTION
**Pull request by AnnoyingRain5**
This should resolve https://github.com/xenia-project/xenia/issues/1579

### CODE REVIEW

**Comment by Triang3l** [comment]
There are some typos, as well as line break formatting different than what is required by our clang-format (run `xb format` after `git add` if you have outstanding changes or checked out a previous commit if you want to replace via force-pushing, or `clang-format -i src/xenia/….cc` directly if you want to format non-staged files). Still, this is a very temporary change, I don't think having such a message is important, considering it's already written on our wiki.

File: src/xenia/app/xenia_main.cc [hunk 1]
```diff
@@ -47,7 +47,10 @@
 #include "third_party/xbyak/xbyak/xbyak_util.h"
 
 DEFINE_string(apu, "any", "Audio system. Use: [any, nop, sdl, xaudio2]", "APU");
-DEFINE_string(gpu, "any", "Graphics system. Use: [any, d3d12, vulkan, null]",
+DEFINE_string(gpu, "any",
+              "Graphics system. Use: [any, d3d12, vulkan, null]\nWARNING: "
+              "Vulkan support is obsolete known to be buggy.\nIt is highly "
```
Comment (Triang3l) [inline src/xenia/app/xenia_main.cc:52]: "obsolete known", "recommen**e**ded" are not correct in English.

File: src/xenia/app/xenia_main.cc [hunk 2]
```diff
@@ -47,7 +47,10 @@
 #include "third_party/xbyak/xbyak/xbyak_util.h"
 
 DEFINE_string(apu, "any", "Audio system. Use: [any, nop, sdl, xaudio2]", "APU");
-DEFINE_string(gpu, "any", "Graphics system. Use: [any, d3d12, vulkan, null]",
+DEFINE_string(gpu, "any",
```
Comment (Triang3l) [inline src/xenia/app/xenia_main.cc:50]:
```
DEFINE_string(
  gpu, "any",
  "long multiline string"
```

**Review by Triang3l — CHANGES_REQUESTED** [review]

File: src/xenia/app/xenia_main.cc (hunk 1 above)
Comment (AnnoyingRain5) [reply src/xenia/app/xenia_main.cc]: I have no idea how I missed that, I have fixed both of the typos in my latest commit.

**Comment by AnnoyingRain5** [comment]
> I don't think having such a message is important, considering it's already written on our wiki.

[This issue](https://github.com/xenia-project/xenia/issues/1579) has been left open and according to Etokapa in discord, 

> There's some text in the titlebar, but many people don't read it and then wonder why Vulkan doesn't work.

So I still believe that this PR should be merged.

**Comment by Triang3l** [comment]
Since we are moving quickly to a new Vulkan-based GPU emulation implementation (#2028), and it's developed on the main branch, this is not actual anymore.

**Pull request closed without merging**